spec/paperclip/attachment_registry_spec.rb

```diff
@@ -54,21 +54,50 @@
         greeter: { ciao: 'greeting' }
       }
       foo = Class.new
-      Paperclip::AttachmentRegistry.register(foo, :avatar, { yo: 'greeting' })
-      Paperclip::AttachmentRegistry.register(foo, :greeter, { ciao: 'greeting' })
+      Paperclip::AttachmentRegistry.register(foo,
+                                             :avatar,
+                                             { yo: 'greeting' })
+      Paperclip::AttachmentRegistry.register(foo,
+                                             :greeter,
+                                             { ciao: 'greeting' })
 
       definitions = Paperclip::AttachmentRegistry.definitions_for(foo)
 
       assert_equal expected_definitions, definitions
     end
 
-    it "produces defintions for subclasses" do
+    it 'produces defintions for subclasses' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.